site2/docs/tiered-storage-aliyun.md

```diff
@@ -18,8 +18,7 @@ Follow the steps below to install the Aliyun OSS offloader.
 
 This example uses Pulsar 2.8.0.
 
-1. [Download the Pulsar tarball](getting-started-standalone.md#install-pulsar-using-binary-release).
-
+1. [Download the Pulsar tarball](getting-started-standalone.md).
 2. Download and untar the Pulsar offloaders package, then copy the Pulsar offloaders as `offloaders` in the Pulsar directory. See [Install tiered storage offloaders](getting-started-standalone.md#install-tiered-storage-offloaders-optional).
```

Comment:
This line and the similar line in `tiered-storage-s3` should be updated. Thus leave this PR as draft for now :)